src/HealthChecks/IetfResponseWriter.cs

```diff
@@ -91,7 +92,10 @@ public static Task IetfResponseWriter(HttpContext httpContext, HealthCheckResult
                 new Dictionary<string, HealthReportEntry> { { CosmosHealthCheck.ServiceId, new HealthReportEntry(res.Status, res.Description, totalTime, res.Exception, res.Data) } },
                 totalTime);
 
-            httpContext.Response.Headers.Add(CpuCounter.CapacityHeader, $"current={CpuCounter.CpuPercent}, target={App.Config.BurstTarget}, max={App.Config.BurstMax}");
```

Comment:
I would encapsulate this bock into a method so it's easy to change.

Maybe put it on CpuCounter? Something like:

CpuCounter.AddBurstHeader(context);